sessions/react-basics/demo-vanilla-js-start/index.js

```diff
@@ -0,0 +1,18 @@
+const rootElement = document.querySelector("[data-js='root']");
+
+function App() {
+  const appContainer = document.createElement("main");
+  appContainer.classList.add("App");
+
+  const button = document.createElement("button");
+  button.type = "button";
+  button.textContent = "click with Vanilla JS ";
+  button.addEventListener("click", () => console.log("Hello Vanilla JS Land"));
+
+  appContainer.append(button);
```

Comment:
**@NikRoe  Good point! Tank you** 

Initially, I had thought about building the button with the students as a recap exercise. However, I decided against it because the session is already quite extensive, and we know that attention can drop quickly during session.

For this reason, I would have the button pre-built as part of the session demo. 

I'll adjust the session guide.